lib/pronto/commentator/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Pronto
   module CommentatorVersion
-    VERSION = '0.1.0'
+    VERSION = '0.1.1'
```

Comment:
http://semver.org/ says that for X.Y.Z, X + Y must change if we change our API behavior - AFAIK, we're not doing this, so we shouldn't change them.

But it also says if we're using the software in prod X should be 1. So... lol

> What should I do if I update my own dependencies without changing the public API?
> That would be considered compatible since it does not affect the public API. Software that explicitly depends on the same dependencies as your package should have their own dependency specifications and the author will notice any conflicts. Determining whether the change is a patch level or minor level modification depends on whether you updated your dependencies in order to fix a bug or introduce new functionality. I would usually expect additional code for the latter instance, in which case it’s obviously a minor level increment.

I think we can do either and it's fine? I'm more worried about how I test circlemator with this change tbh.